client/src/components/v2/CameraRecording.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Card } from 'react-bootstrap';
+import Button from 'react-bootstrap/Button';
+import { emit } from 'api/v2/socket';
+import PropTypes from 'prop-types';
+import CameraRecordingStatus from './CameraRecordingStatus';
+
+/**
+ * @typedef {object} CameraRecordingProps
+ * @property {string} devices Comma separated values of devices
+ */
+
+/**
+ * Camera recording buttons for starting and stopping video recording.
+ *
+ * Starts/stops recording for both displays.
+ *
+ * This is a feature intended for V3 but is current in V2 for testing.
+ *
+ * @param {CameraRecordingProps} props Props
+ * @returns {React.Component<CameraRecordingProps>} Component
+ */
+export default function CameraRecording({ devices }) {
+  const startRecording = () => {
+    emit('start-camera-recording');
+  };
+
+  const stopRecording = () => {
+    emit('stop-camera-recording');
+  };
+
+  return (
+    <Card>
+      <Card.Body>
+        <Card.Title>Recording Controls</Card.Title>
+        {
+          devices.split(',').map((device) => (
```

Comment:
You can pass in an array of strings for devices here to make the format of the argument clearer and reduce the overhead of splitting.

Reply:
Done
https://github.com/monash-human-power/dashboard/pull/74/commits/7c9a27d57887c765470f6eaf8879817115defbc9

I should have known that React would have a way to pass objects as props instead of just a string haha.